tests/generation/test_utils.py

```diff
@@ -1815,22 +1817,14 @@ def test_generate_compile_fullgraph(self):
             }
 
             for model_inputs in input_ids_sets:
-                # dynamic cache
+                # eager dynamic cache
                 output_dynamic = model.generate(model_inputs, **generation_kwargs)
 
-                # eager static cache
-                torch.compiler.reset()
-                model.generation_config.cache_implementation = "static"
-                output_static = model.generate(model_inputs, **generation_kwargs)
-                self.assertListEqual(output_dynamic.tolist(), output_static.tolist())
-
-                # compiled static cache (removes the cache initialized in the previous check, to confirm we can
-                # initialize the cache in full compiled mode)
-                model._cache = None
+                # end-to-end compiled dynamic cache
```

Comment:
the reference for end-to-end compilation is with dynamic cache -- I forgot to update this test before the last set of changes in the PR that introduced end-to-end compilation :)

Reply:
this includes recompile no? Or dynamic shapes are handled now?

Reply:
Yes, it issues a recompile

(but static cache doesn't work at the moment -- both things, recompilations and using static caches, need to be addressed)